cypress/support/commands.js

```diff
@@ -24,17 +24,18 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
-Cypress.Commands.add('login', (email, username, password) => {
+Cypress.Commands.add('registrationAndLogin', (email, username, password) => {
```

Comment:
why did you need to change the name of that method?)

Reply:
fixed